crates/router/Cargo.toml

```diff
@@ -42,6 +42,7 @@ actix-web = "4.3.1"
 async-bb8-diesel = { git = "https://github.com/jarnura/async-bb8-diesel", rev = "53b4ab901aab7635c8215fd1c2d542c8db443094" }
 argon2 = { version = "0.5.0", features = ["std"] }
 async-trait = "0.1.68"
+aws-config = { version = "0.55.3", optional = true }
```

Comment:
Do we need this dependency in the `router` crate?

Reply:
yes we are using it